src/response/app_keys_list.rs

```diff
@@ -0,0 +1,101 @@
+use std::prelude::v1::*;
+
+use std::fmt;
+
+use serde::{
+    de::{Deserializer, MapAccess, Visitor},
+    Deserialize,
+};
+
+use crate::{
+    application::{AppId, AppKey},
+    credentials::ServiceId,
+};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AppKeysList {
+    service: Option<ServiceId>,
```

Comment:
Maybe calling these service_id and app_id would be more accurate.

Reply:
Yes, it would and I prefer that as well.